### PR TITLE
Packageretrieve

### PR DESCRIPTION
**Pull request by erocheleau**
Retrieve the unpackaged files described in the package.xml selected by the user at the moment of retrieve. Unzips the result of the retrieve command in the same folder where you selected the package.xml file.

To test this, simply run the extension in a folder where you have a package.xml somewhere describing metadata objects you have in your connected org and it should ask you to choose your package.xml if you have more than one, otherwise it will take the only one it finds.
It should created the folder structure related to the objects you have in your package.xml and then unzip those files in the proper folders.

This looks like a lot of changes because of the linting of files at the same time.

[x] Code is up-to-date with the `master` branch

### CODE REVIEW

**Review by lbenie — APPROVED** [review]

**Review by tedre191 — CHANGES_REQUESTED** [review]

File: package.json [hunk 1]
```diff
@@ -1,174 +1,169 @@
...
+            {
+                "command": "extension.contextualQuery",
+                "title": "Contextual Query"
+            },
+            {
+                "command": "extension.executeCode",
+                "title": "Execute APEX code"
+            },
+            {
+                "command": "extension.sfdeploy",
```
Comment (erocheleau) [inline package.json]: I did not do dis

File: src/commands/retrieveCommand.ts [hunk 2]
```diff
@@ -14,59 +14,47 @@ import * as utils from '../utils';
...
-  /**
-   * Implements execute from {@link Command}
-   *
-   * Executes the command for this class
-   */
+	/**
+	 * Implements execute from {@link Command}
+	 *
+	 * Executes the command for this class
+	 */
```
Comment (erocheleau) [inline src/commands/retrieveCommand.ts]: Should be fixed, dont know how to mark this as resolved...

File: src/commands/retrieveCommand.ts [hunk 3]
```diff
@@ -14,59 +14,47 @@ import * as utils from '../utils';
...
-
   private output: vscode.OutputChannel;
   private retrieveTarget: string;
 
-  /**
-   * Creates a Retrieve command
-   */
+	/**
+	 * Creates a Retrieve command
+	 */
```
Comment (erocheleau) [inline src/commands/retrieveCommand.ts]: Should be fixed, dont know how to mark this as resolved...

File: package.json (hunk 1 above)
Comment (lbenie) [inline package.json]: my fault 😢  hehe bad copy paste 😈

**Comment by erocheleau** [comment]
:worksonmymachine:
🚀 

Same Travis error as before even if I merged master into my branch.

`error: unable to create symlink tslint.json (File name too long)`

**Comment by lbenie** [comment]
Hmm that issue has been resolved...

imo just merge it and it should be fine

**Review by tedre191 — APPROVED** [review]

**Pull request merged**